### PR TITLE
adding script for super-resolution from a stack of jpegs shot handheld.

### PR DESCRIPTION
**Pull request by isaacullah**
Script and readme file inside the "superres" directory.

### CODE REVIEW

**Comment by patdavid** [comment]
Thank you for the addition, @isaacullah!  Just pinging @paperdigits to let him know this got merged as well.

**Comment by isaacullah** [comment]
Sure thing! Tied up teaching classes today, but will make a post in the forum about this soon..

**Comment by paperdigits** [comment]
Awesome! Thanks for your contribution.

File: superres/SR.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (isaacullah) [inline superres/SR.sh]: Thanks for noticing that! Yes, it's an error. Should be `#!/bin/sh`. I will fix and re-push.

File: superres/SR.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+## This is a script to create a 'super resolution' image from a stack of lower resolution images
+
+mkdir -p resized # make a new directory to hold resized images
+convert *.jpg -resize 200% ./resized/%04d.jpg # upscale images by 200% (4x more pixels) and copy to new directory
+cd ./resized
+align_image_stack -a al -C -t 0.3 -c 20 -v *.jpg # auto-align resized images and crop all of them to aligned area
+cd ..
+convert ./resized/al* -evaluate-sequence mean SR_mean.jpg # calculate average at each new pixel
+convert ./resized/al* -evaluate-sequence median SR_median.jpg # calculate median at each new pixel
+
```
Comment (isaacullah) [inline superres/SR.sh]: Just a scripting convention I acquired early on. I was taught that it's more readable as it makes it clear that the relative path is based from the current working directory. It has same functionality as a relative path with no leading slash...

File: superres/SR.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+## This is a script to create a 'super resolution' image from a stack of lower resolution images
+
+mkdir -p resized # make a new directory to hold resized images
```
Comment (isaacullah) [inline superres/SR.sh]: Good idea. I will add it.

File: superres/SR.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+convert *.jpg -resize 200% ./resized/%04d.jpg # upscale images by 200% (4x more pixels) and copy to new directory
+cd ./resized
```
Comment (isaacullah) [inline superres/SR.sh]: Ditto to above. I will add this.

**Pull request merged**